halo1/model.py

```diff
@@ -334,9 +344,40 @@ def import_halo1_region_from_jms(jms, *, name="unnamed", scale=1.0, region_filte
 	scene = bpy.context.collection
 	scene.objects.link(region_obj)
 
+	# If the function was supplied with an armature object attempt to skin to it.
+
+	region_obj.parent = armature_obj
```

Comment:
What happens if the supplied `armature_obj` isn't actually an armature? Do we just parent it anyway, as if it were a standard node?